src/bench/base58.cpp

```diff
@@ -13,10 +13,10 @@
 
 static void Base58Encode(benchmark::State& state)
 {
-    unsigned char buff[32] = {
+    unsigned char buff[33] = {
```

Comment:
A NUL terminator is unnecessary here, as the array is not interpreted as a string.